.circleci/config.yml

```diff
@@ -16,7 +16,7 @@ jobs:
 
       - run:
           name: Concatenate all package-lock.json files into single file for checksum
-          command: cat package.json api/package.json authorizer/package.json > combined-package-lock.txt
+          command: cat package.json api/package.json > combined-package-lock.txt
```

Comment:
It was wrong in the first place (probably me!) but we should use the package-lock.json files rather than package.json